guides/sequential_model.py

```diff
@@ -254,7 +254,7 @@
 - Save your model to disk and restore it. See our
 [guide to serialization & saving](/guides/serialization_and_saving/).
 - Speed up model training by leveraging multiple GPUs. See our
-[guide to multi-GPU and distributed training](/guides/distributed_training).
```

Comment:
In this case the correct URL needs to be absolute: `https://keras.io/guides/distributed_training/` (it isn't on the TF.org website)